docs/wiki/99 - Developers/SQL Dictionary.md

```diff
@@ -47,3 +47,37 @@
 	| `OptimumMaxDays `    | Candidate's maximum age in days for visit to be flagged as _within **optimal** parameters of the study_   | The only effect of this field is a YES/NO flag showing up on the instrument_list page|
 	| `WindowMidpointDays `| Candidate's ideal age in days for a visit                                                                 |  |
 	
+
+- Table: `Project`
+ 
+ 	*This table stores the list of projects configured for the study.*
+ 
+	|         Field        |                  Description                  |                                        Notes                                                 |
+	|:--------------------:|:---------------------------------------------:|:--------------------------------------------------------------------------------------------:|
+	| `ProjectID`          | Identifier of the project                     | Avoid setting this field explicitly when inserting data, it auto increments.                 |
+	| `Name`               | Full name of the project                      |                                                                                              |
+	| `recruitmentTarget`  | Expected number of candidates to be recruited |                                                                                              |
+
+
+- Table: `subproject`
+ 
+ 	*This table stores the list of subprojects configured for the study.*
+ 
+	|        Field       |                  Description                  |                                     Notes                                    |
+	|:------------------:|:---------------------------------------------:|:----------------------------------------------------------------------------:|
+	| `SubprojectID`     | Identifier of the subproject                  | Avoid setting this field explicitly when inserting data, it auto increments. |
+	| `title`            | Name of the subproject                        |                                                                              |
+	| `useEDC`           | Use the Expected date Of Confinement          |                                                                              |
+	| `WindowDifference` |                                               | Deprecated ??                                                                |
```

Comment:
This was used to determine if the error flagging for the candidate age on an instrument should be based on the 'optimal' window in the visit_windows table, or if it should be done by guessing what the nearest test battery is and assuming that it should have been in that window. As far as I know the code hasn't been removed or deprecated.

Reply:
i doont see it being used anywhere. can you point me to the code ?

Reply:
I can't find it, I guess you can leave it marked as deprecated. Can you remove the `??`?